src/mj_utils_merge_mujoco_models.cpp

```diff
@@ -87,6 +87,17 @@ static void merge_mujoco_option(const std::string & fileIn, const pugi::xml_node
   }
 }
 
+static void merge_mujoco_extension(const std::string & fileIn, const pugi::xml_node & in, pugi::xml_node & out)
+{
+  merge_mujoco_node("extension", fileIn, in, out);
+  const auto & plugin = in.child("plugin");
```

Comment:
This code considers only the first plugin and ignores the subsequent plugins if more than one is present. I am not familiar with pugixml and could you please help me how to fix it? @gergondet

Reply:
Do you have examples with the use of the `extension` tag? Possibly one that highlights the issue when multiple plugins are used. Is there multiple `extension` tag in the file?